package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "build": "tsc",
     "lint:ci": "eslint --report-unused-disable-directives .",
-    "lint": "eslint --report-unused-disable-directives --fix .",
+    "lint": "eslint --report-unused-disable-directives .",
```

Comment:
`--report-unused-disable-directives` should be removed as this flag no longer exists (or is deprecated?).

I don't feel too strongly about `--fix` except that we have it set up with `--fix` in most of our other repos.

Reply:
I was thrown off by running lint changing my source files